src/website/views/components/modal.pug

```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
         </div>
       </div>
     </div>
+
+h4 Scrollable
```

Comment:
Can you move this Scrollable modal example above the simple modal example?

Reply:
done